changes/533.feature

```diff
@@ -0,0 +1 @@
+Wrap aiotools persistent group under asyncio.shield
```

Comment:
"wrap ~ with" would be more correct English instead of "wrap ~ under".
The title and changelog should include a short description on "what the PR improves" by "how the PR changes the code", not only about the too specific code modification.